src/Microsoft.OData.Edm/Csdl/CsdlJsonWriter.cs

```diff
@@ -100,13 +150,43 @@ private void WriteSchemata()
             }
         }
 
+        /// <summary>
+        /// Schema Object asynchronously

```

Comment:
```suggestion
        /// writes schema objects asynchronously

```

Reply:
@ElizabethOkerio This is done across the other methods.